libs/functions/calculate_projections.py

```diff
@@ -80,21 +87,25 @@ def _calculate_projection_data(file_path):
         peak_hospitalizations_short_falls = df.iloc[df.all_hospitalized.idxmax()].short_fall
         peak_deaths_date = df.iloc[df.new_deaths.idxmax()].date
         population = df.iloc[0].population
-        return [
-            hosp_16_days,
-            hosp_32_days,
-            short_fall_16_days,
-            short_fall_32_days,
-            mean_hospitalizations,
-            mean_deaths,
-            peak_hospitalizations_date,
-            peak_deaths_date,
-            hospitals_shortfall_date,
-            peak_hospitalizations_short_falls,
-            beds_at_peak_hospitalization_date,
-            population
-        ]
-    return None
+
+        record['State'] = state
+        if fips:
+            record['FIPS'] = fips
+
+        record['16-day_Hospitalization_Prediction'] = hosp_16_days
+        record['32-day_Hospitalization_Prediction'] = hosp_32_days
+        record['16-day_Beds_Shortfall'] = short_fall_16_days
+        record['32-day_Beds_Shortfall'] = short_fall_32_days
+        record['Mean Hospitalizations'] = mean_hospitalizations
+        record['Mean Deaths'] = mean_deaths
+        record['Peak Hospitalizations On'] = peak_hospitalizations_date
+        record['Peak Deaths On'] = peak_deaths_date
+        record['Hospital Shortfall Date'] = hospitals_shortfall_date
+        record['Peak Hospitlizations Shortfall'] = peak_hospitalizations_short_falls
+        record['Beds at Peak Hospitilization Date'] = beds_at_peak_hospitalization_date
+        record['Population'] = population
+
+    return pd.Series(record)
```

Comment:
It feels weird that this is returning a series.  I do love these having keys, but doesn't feel like it needs to be a series

Reply:
Pandas recomends returning series when build dataframes as a result of an apply